src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
```

Comment:
Definitely a good point, that'll work then. I do think we should indicate that the unit is _minutes_ because people will tend to assume it's a length of distance not time. That could be done in the display values also.